haystack/preprocessor/utils.py

```diff
@@ -32,11 +32,13 @@ def eval_data_from_file(filename: str) -> Tuple[List[Document], List[Label]]:
 
     with open(filename, "r") as file:
         data = json.load(file)
+        if "title" not in data["data"][0]:
+            logger.warning(f"No title information found for documents in QA file: {filename}")
         for document in data["data"]:
             # get all extra fields from document level (e.g. title)
             meta_doc = {k: v for k, v in document.items() if k not in ("paragraphs", "title")}
             for paragraph in document["paragraphs"]:
-                cur_meta = {"name": document["title"]}
+                cur_meta = {"name": document.get("title","missing")}
```

Comment:
We should rather use an empty string or `None` as the fallback option instead of "missing". 
The title can play a big role when creating the embeddings via DPR and we will easily treat "missing" as a real title.